packages/editor/src/components/collab-sidebar/index.js

```diff
@@ -22,7 +22,7 @@ import { store as editorStore } from '../../store';
 import AddCommentButton from './comment-button';
 import AddCommentToolbarButton from './comment-button-toolbar';
 
-const threadsEmptyArray = [];
+const EMPTY_ARRAY = [];
```

Comment:
This matches the name used in other places and makes it easier to search the code base.